docs/product/alerts/create-alerts/metric-alert-config.mdx

```diff
@@ -132,11 +132,11 @@ Change alerts, or alerts that use a percent change threshold, are useful when yo
 <Include name="feature-available-for-plan-trial-business.mdx" />
 <Include name="feature-available-for-user-group-early-adopter.mdx" />
 
-Anomaly alerts can be used when you don't know what threshold to set, but you know you want to be alerted when something is far outside the bounds of normalcy. Sentry will look at the historical data for the given metric and determine if the current data is anomalous. You have three options for the responsiveness of the alert: low, medium, and high. Low responsiveness means the alert will fire less frequently with each instance being higher signal. High responsiveness will capture more instances of the alert, which creates more coverage with a greater likelihood for false positives. Our recommendation is to start with our default (medium), and iterate based on the results you see.
-Another setting is to set the direction of the alert: above and/or below the expected bounds. This setting can help increase the signal of your alert rule.
+Anomaly alerts automatically detect trends outside expected values. They can be especially helpful for spiky or seasonal data that are too noisy for static or percentage based thresholds. Sentry will look at historical data for the given metric and determine if the current data is anomalous. Apdex is currently not supported for anomaly alerts.
 
-Critical and Resolved thresholds will be controlled by Sentry rather than the user. When no more anomalies are detected, the alert will resolve. Anomaly alerts are not available for all metric alert types.
+You have three options for the responsiveness of the alert: low, medium, and high. Low responsiveness means the alert will fire less frequently with each instance being higher signal. High responsiveness will capture more instances of the alert, which creates more coverage with a greater likelihood for false positives. Our recommendation is to start with our default (medium), and iterate based on the results you see.
```

Comment:
> with each instance being higher signal

What does this mean?

Reply:
each instance of the alert. would replacing instance with "time" make more sense?

Reply:
I guess I don't understand why the alert firing multiple times at a low responsiveness means a higher signal

Reply:
since each time it fires, it has to be a lot more confident....would changing the word to "higher confidence" make more sense?

Reply:
Ohh do you mean that between low, medium, and high they have higher confidence? The way I read this was that each time a _low_ confidence alert fires it has more confidence somehow

Reply:
@ceorourke - I modified the text to: "You have three options for the responsiveness of the alert: low, medium, and high. Low responsiveness means the alert will fire less frequently with each instance fired more likely to be a problem. High responsiveness will capture more instances of the alert, which creates greater coverage with a higher likelihood for false positives. Our recommendation is to start with our default (medium), and iterate based on the results you see."

Thoughts on whether that's clear?

Reply:
You could consider this:
"There are three options for alert responsiveness: low, medium, and high. If you choose low responsiveness, then your alert will fire less frequently but with higher confidence (i.e. each time your alert fires, it is more likely to be a problem). If you choose high responsiveness, then your alert will fire more frequently with a greater chance of false positives. Our recommendation is to start with medium responsiveness, our default, and adjust the responsiveness based on the results you see."

Reply:
LGTM, let's ship that